app/config/permissions.neon

```diff
@@ -168,7 +175,7 @@ permissions:
           - viewGroups
 
     - allow: true
-      role: supervisor
+      role: supervisor-student
```

Comment:
I don't think a supervisor-student should do anything with instances. Any counterexamples?